### PR TITLE
[UHN-MA-52] Notification reducer

### PR DESCRIPTION
**Pull request by Asiapenguin**
- Using redux to store notification data
  - For payload, please look at [this](https://github.com/CodetheChangeFoundation/UHN-Backend-Server/pull/58)
- Please let me know if the redux implementation is not correct, I am not that familiar with the syntax of it. It seems to work.

### CODE REVIEW

File: store/actions/NotificationActions.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { SET_NOTIFICATION } from "./Types";
+
+export const setNotification = (notification) => {
+  return {
+    type: SET_NOTIFICATION,
+    data: {
+      notification
```
Comment (ad2969) [inline store/actions/NotificationActions.js]: Perhaps turn into array so we can consider multiple notifications

**Comment by Asiapenguin** [comment]
I havent implemented the removal of notification entries yet. Both at logout and at notification resolved state. Should this be a separate issue? @mxnyee and @ad2969, wanna hear your thoughts

**Comment by mxnyee** [comment]
Shouldn't the notification entries be removed from the array once the user hits either "on my way" or "can't help now"? In that case, I'd be removing them in https://github.com/CodetheChangeFoundation/UHN-Mobile-App/issues/67

**Comment by Asiapenguin** [comment]
But at the directions screen, the responder still needs the help request id to update the help request status to arrived

**Comment by mxnyee** [comment]
Yeah true. How does a help request get marked as resolved?

**Comment by Asiapenguin** [comment]
There is no mockup for that screen. I am imagining it to be similar to the directions screen. This got brought up when Tai was creating the list of data analytics he wanted for tracking.

**Comment by Asiapenguin** [comment]
Merging for now. Note: remember to remove entries in notifications.

**Pull request merged**